reliability-engineering/pipelines/tasks/concourse-land-workers.yml

```diff
@@ -36,9 +38,11 @@ run:
     function get_old_worker_count() {
       wc -l <workers/names
     }
-    while [ "$(get_current_worker_count)" = "$(get_old_worker_count)" ]; do
-      echo "...still only $(get_old_worker_count) workers"
+    retrys=0
+    until [ "$(get_current_worker_count)" != "$(get_old_worker_count)" ] || [ $retrys -gt 50 ]; do
+      echo "...still only $(get_old_worker_count) workers, retry attempt $retrys"
       sleep 3
+      retrys=$((retrys+1))
     done
     echo "OK!"
```

Comment:
It might be nicer to at least print a different message if it exceeds the retries. Or print the the value of `get_current_worker_count` when it succeeds...